v2/features/farms/FarmsTableRowHeader.tsx

```diff
@@ -160,24 +183,31 @@ const FarmsTableRowHeader: FC<Props> = ({ jar, simple, open }) => {
           "rounded-tl-xl flex items-center",
         )}
       >
-        <div className="w-9 h-9 rounded-full border-3 border-gray-outline mr-3">
-          <Image
-            src="/alchemix.png"
-            className="rounded-full"
-            width={200}
-            height={200}
-            layout="responsive"
-            alt={jar.depositToken.name}
-            title={jar.depositToken.name}
-          />
-        </div>
+        <TokenIcon
```

Comment:
I was really hoping, for this issue, that you'd try to create a brand new `LpIcon` type class or object, which is based on pfcore. 

For example, pass in a JarDefinition. The JarDefinition has a few key parts:  

1) The protocol. This could be the bigger image in the back
2) a deposit token, which has a set of 'components'

If the jar has one component (single-staking), you could overlay the single image on the bottom-right of the protocol image. 
If the jar has two components (LP), you would overlay the two component images. 
If the jar has three components (tricrypto etc), you would arrange 3 components in a triangle on the bottom right. 
etc etc etc. 

Re-using image code from v1 is risky.

Reply:
Agree with @robstryker here. You might have noticed that the way I wrote the initial v2 code is so that one fine day we can completely delete everything outside the v2 folder. So even if a component can be reused to a large extent, I think it's better to find an appropriate location for it in v2 and copy the code over, do any cleanup there.

However, I think it's ok to get this out as it is. I can work on this in a follow-up PR. I think it makes sense for you guys to spend your efforts where you are the most productive (pf-core-related code) while I work on the new frontend components (Tailwind/Next have a learning curve).